.storybook/components/DesignTokens/DesignTokens.css

```diff
@@ -76,4 +76,3 @@
 .sg-animation-demo-box:hover {
   background-color: #dddddd;
 }
-/*# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIkRlc2lnblRva2Vucy5jc3MiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUE7RUFDRSxhQUFFO0VBQ0YsY0FBRTtFQUNGLDREQUFFO0VBQ0YsZ0JBQUU7RUFDRixTQUFFO0VBQ0YsVUFBRTtBQUNKO0FBQ0E7RUFFRSxtQkFBRTtFQUFGLFVBQUE7RUFDQSxjQUFFO0VBQ0YsdUJBQUU7RUFDRixjQUFFO0VBQ0Ysc0JBQUU7RUFDRixrQkFBRTtBQUNKO0FBQ0E7RUFDRSxjQUFFO0VBQ0YsV0FBRTtFQUNGLG9CQUFFO0VBQ0Ysa0JBQUU7QUFDSjtBQUNBO0VBQ0UsZUFBRTtBQUNKOztBQUVBO0VBQ0UsZUFBZTtFQUNmLGlCQUFpQjtFQUNqQixrQkFBa0I7RUFDbEIsV0FBVztFQUNYLG1CQUFtQjtFQUNuQixxQkFBcUI7QUFDdkI7O0FBRUE7RUFDRSxpQkFBaUI7QUFDbkI7QUFDQTtFQUNFLGlCQUFpQjtBQUNuQjs7QUFFQTtFQUNFLGlCQUFpQjtBQUNuQjs7QUFFQTtFQUNFLGlCQUFpQjtBQUNuQjs7QUFFQTtFQUNFLGlCQUFpQjtFQUNqQixrQkFBa0I7QUFDcEI7O0FBRUE7RUFDRSxjQUFBO0VBQ0EsZUFBYTtFQUNmLHlCQUFBO0FBQ0E7O0FBRUE7RUFDRSx5QkFBVztFQUNYLG1CQUFBO0VBQ0Ysa0JBQUE7RUFDQSxhQUFBO0FBQ0E7O0FBRUE7RUFDQSxXQUFBO0VBQ0UseUJBQXlCO0VBQ3pCLGVBQWU7QUFDakI7O0FBRUE7RUFDRSx5QkFBeUI7QUFDM0IiLCJmaWxlIjoiRGVzaWduVG9rZW5zLmNzcyIsInNvdXJjZXNDb250ZW50IjpbIi5zZy1jb2xvcnN7ZGlzcGxheTpncmlkO2dyaWQtZ2FwOjEwcHg7Z3JpZC10ZW1wbGF0ZS1jb2x1bW5zOnJlcGVhdChhdXRvLWZpbGwsIG1pbm1heCgyODBweCwgMWZyKSk7bGlzdC1zdHlsZTpub25lO21hcmdpbjowO3BhZGRpbmc6MH0uc2ctY29sb3JzIGxpey13ZWJraXQtYm94LWZsZXg6MTstbXMtZmxleDphdXRvO2ZsZXg6YXV0bztwYWRkaW5nOjAuM2VtO21hcmdpbjowIDAuNWVtIDAuNWVtIDA7bWluLXdpZHRoOjVlbTtib3JkZXI6MXB4IHNvbGlkICNkZGQ7Ym9yZGVyLXJhZGl1czo4cHh9LnNnLXN3YXRjaHtkaXNwbGF5OmJsb2NrO2hlaWdodDo0ZW07bWFyZ2luLWJvdHRvbTowLjNlbTtib3JkZXItcmFkaXVzOjVweH0uc2ctbGFiZWx7Zm9udC1zaXplOjE0cHh9XG4iXX0= */
```

Comment:
`yarn lint:fix` removed this line of code

Reply:
Nope, safe to remove!